ambari-server/src/main/resources/stack-hooks/before-ANY/scripts/params.py

```diff
@@ -65,6 +65,7 @@
 
 ambari_server_hostname = execution_command.get_ambari_server_host()
 
+stack_version_unformatted = execution_command.get_mpack_version()
```

Comment:
Can you check if we are using stack_version_unformatted anywhere and get rid of it if not?
@jayush , do we need to keep it for any legacy stacks?

Reply:
yes, it is used, I once removed it,but it was invoked in other script